NOTE: Might need to block the addition in the first place:
https://android.googlesource.com/platform/external/bazelbuild-rules_python/+/1508c18d9e27325a8d9c574a90d205e0a1d8a78f/python/pip_install/tools/wheel_installer/wheel.py#240

diff --git python/private/pypi/whl_installer/wheel_installer.py python/private/pypi/whl_installer/wheel_installer.py
index c6c2961..b0f9f76 100644
--- python/private/pypi/whl_installer/wheel_installer.py
+++ python/private/pypi/whl_installer/wheel_installer.py
@@ -129,7 +129,9 @@ def _extract_wheel(

     extras_requested = extras[whl.name] if whl.name in extras else set()

-    dependencies = whl.dependencies(extras_requested, platforms)
+    dependencies = whl.dependencies(extras_requested, platforms)
+    if (whl.name == "triton") and ("torch" in dependencies):
+        dependencies.remove("torch")

     with open(os.path.join(installation_dir, "metadata.json"), "w") as f:
         metadata = {

